test/conformance/go.mod

```diff
@@ -1,29 +1,31 @@
 module github.com/ovn-org/ovn-kubernetes/test/conformance
 
-go 1.20
+go 1.21
+
+toolchain go1.21.7
```

Comment:
why is there a toolchain here?

I think its because you have not updated net-pol-api itself I think you can drop this commit from this PR and I'll get it in via https://github.com/ovn-org/ovn-kubernetes/pull/4245/commits/c918793782dc87886dad1fe2fa404544ce1b5ac2 ? wdyt?

Reply:
yeah i agree, will drop the commit